src/babashka/neil.clj

```diff
@@ -140,18 +140,20 @@
   - alias-kw - alias keyword, like :kaocha or :dev
   - alias-map - string representing alias contents
   "
-  [deps-file-str alias-kw alias-map]
+  [deps-file-str alias-kw alias-map-str]
   (let [edn-nodes (edn-nodes deps-file-str)
         edn (edn/read-string deps-file-str)
-        existing-aliases (get-in edn [:aliases])]
+        existing-aliases (get-in edn [:aliases])
+        alias-map (edn/read-string alias-map-str)]
+    (assert (map? alias-map) "Alias map must be a map")
     (if-not (get existing-aliases alias-kw)
       (let [node-with-aliases-key (if-not (seq existing-aliases)
                                     (r/assoc edn-nodes :aliases {})
                                     edn-nodes)
             node-with-new-alias (reduce (fn [node [k v]]
                                           (r/assoc-in node [:aliases alias-kw k] v))
                                         node-with-aliases-key
-                                        (edn/read-string alias-map))]
+                                        (into (sorted-map) alias-map))]
```

Comment:
I didn't like that the order of the keys in the alias map would be arbitrary, so I sorted the keys.